### PR TITLE
DEV-4183 numerical-operation-the-return

### PR DESCRIPTION
**Pull request by mikysett**
new exercise for scripting piscine

### CODE REVIEW

File: subjects/devops/numerical_operations_the_return/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import numerical_operations_the_return
+
+print(numerical_operations_the_return.modulo(10, 3))
+print(numerical_operations_the_return.divide(10, 3))
+print(numerical_operations_the_return.divide(10, 0))
+print(numerical_operations_the_return.integer_division(10, 3))
+```
+
+```bash
+$ python test.py
```
Comment (MSilva95) [inline subjects/devops/numerical_operations_the_return/README.md:29]: python -> python3

**Review by MSilva95 — CHANGES_REQUESTED** [review]
just add the python 3 and its good to go

File: subjects/devops/numerical_operations_the_return/README.md (hunk 1 above)
Comment (mikysett) [reply subjects/devops/numerical_operations_the_return/README.md]: done

**Review by MSilva95 — APPROVED** [review]
LGTM!

**Pull request merged**